lib/std/fmt.zig

```diff
@@ -1582,7 +1582,8 @@ test parseInt {
     try std.testing.expectEqual(@as(i5, -16), try std.fmt.parseInt(i5, "-10", 16));
 }
 
-fn parseIntWithSign(
+/// Like `parseIntWithGenericCharacter`, but with a sign argument.
+pub fn parseIntWithSign(
```

Comment:
This is a generally useful function IMO, and I needed access to it from the ZON parser, so I made it public